txpool/blocklist_test.go

```diff
@@ -0,0 +1,169 @@
+package txpool
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vechain/thor/v2/thor"
+)
+
+func SetupFile(t *testing.T, testFilePath string, dummyData string) {
+	err := os.WriteFile(testFilePath, []byte(dummyData), 0644)
+	if err != nil {
+		t.Fatalf("Failed to create test file: %s", err)
+	}
+}
+
+func TestLoad(t *testing.T) {
+
```

Comment:
Is this new empty line on purpose?  I saw serval functions has a new empty line.